src/jit/emitfmtsxarch.h

```diff
@@ -122,6 +122,7 @@ IF_DEF(MRD_OFF,     IS_GM_RD,                   DSP)      // offset mem
 IF_DEF(RRD_MRD,     IS_GM_RD|IS_R1_RD,          DSP)      // read   reg , read [mem]
 IF_DEF(RWR_MRD,     IS_GM_RD|IS_R1_WR,          DSP)      // write  reg , read [mem]
 IF_DEF(RRW_MRD,     IS_GM_RD|IS_R1_RW,          DSP)      // r/w    reg , read [mem]
+IF_DEF(RRW_MRD_CNS, IS_GM_RD|IS_R1_RW,          DSP_CNS)  // r/w    reg , read [mem], const
```

Comment:
The `R_R_I` instructions define and use `RRW_RRW_CNS` and I mirrored that here (but for mem and stk).

I actually expected the first register to be `RWR` (write, rather than read/write). Does anyone know why it was set as read/write? (maybe to do with AVX support...)

Reply:
It's not clear to me either why this is this way.